qcsrc/misc.qc

```diff
@@ -244,7 +244,7 @@ void() fire_fly =
 
 void() fire_touch =
 {
-	T_Damage (other, self, self, 200);
+	T_Damage (other, self, self, 20);
```

Comment:
Is this a fix? Is it supposed to be 20?

Reply:
I checked both the default QW progs and NQ progs and they both have 20 for this value. I don't know how it got to be 200 in our progs.